CMakeLists.txt

```diff
@@ -254,6 +257,7 @@ check_symbol_exists(BIO_meth_get_ctrl "openssl/bio.h" HAVE_BIO_METH_GET_CTRL)
 check_symbol_exists(BIO_meth_get_create "openssl/bio.h" HAVE_BIO_METH_GET_CREATE)
 check_symbol_exists(BIO_meth_get_destroy "openssl/bio.h" HAVE_BIO_METH_GET_DESTROY)
 check_symbol_exists(DH_get_2048_256 "openssl/dh.h" TS_USE_GET_DH_2048_256)
+check_symbol_exists(SSL_CTX_set_keylog_callback openssl/ssl.h TS_HAS_TLS_KEYLOGGING)
```

Comment:
should this be a quoted include file (like all the previous lines) or unquoted (like the next line)?

Reply:
Great question. In this case it's a matter of style. Since the CMake documention has them quoted, and because @cmcfarlen quoted most of them, I suggest we quote them all. Also, if there happened to be a header with a space in the name, then the quotes would be necessary. I hope that never happens, but theoretically that would be a reason to prefer quotes.